packages/datacommons-mcp/datacommons_mcp/server.py

```diff
@@ -189,18 +186,20 @@ async def validate_child_place_types(
 
 @mcp.tool()
 async def get_available_variables(
-    place_name: str = "world", category: str = "statistics"
+    place_name: str = "world", indicator_desc: str = "statistics"
 ) -> dict:
     """
-    Gets available variables for a place and category.
+    Gets available variables for a place and indicator (category or variable).
```

Comment:
I think we should try to be more consistent with what we are calling things across tools (categories vs topics vs indicators vs stat vars vs groups). We may want to spend a quick sync discussing which verbiage to expose to the agent moving forward.